project-reports/2024/2024-annual-Hyperledger-Besu.md

```diff
@@ -0,0 +1,43 @@
+
+# Goals
+
+## Performance Against Prior Goals
+The Besu maintainers successfully delivered the Cancun hard fork on Ethereum Mainnet and are looking forward to scoping the Prague fork. The Besu project has struggled to court sets of new maintainers from organizations building both public and private networks, but we are hopeful the new financial services working group may yield new interest and contributions. We have made some progress on standardizing features across both network types, like shipping snap-sync as a server. This means that, shortly, private networks and Mainnet can use the same sync methodology (and we may remove fast-sync). We are still working on Bonsai Archive and other feature consolidation efforts. 
+
+The enterprise and public roadmaps were both delivered on the [wiki.](https://wiki.hyperledger.org/pages/viewpage.action?pageId=24781786). We are hopping this will help with adoption, clarity among developers and orgs using Besu, and will keep up the transparency efforts by Maintainers. 
+
+
+## Next Year's Goals
+
+This year the Maintainers are seeking to continue feature consolidation efforts as well as a testing overhaul to advance the quality of the code-base. This will also incorporate modularity efforts that were punted from last years roadmap. 
+
+Modularity and feature consolidation efforts seek to increase code-base quality and encourage contributions by alleviating confusing and obfuscated code-paths for the various use-cases. [New testing efforts seek](https://wiki.hyperledger.org/display/BESU/Testing+Taskforce+Brainstorming) to keep the this code-base quality high, as there have been several release interventions over the last year due to late-introduced bugs that the team wants to desperately avoid to keep velocity high. There are also more upstream users of Besu that will be impacted than ever before (see above).
+
+This year there is also a renewed push around revitalizing and modernizing the plug-in API. This includes efforts by the Consensys Documentation team and developers to revitalize the docs and add tutorials, templates, and much more information with the goal of encouraging the use of the API. As this API was used to build the Linea project's key components, the team hopes to capitalize on these examples to encourage the use of the plug-ins for more novel use-cases like other L2s, private networks, and more creativity that hasn't been given the proper chance quite yet. 
+
+The Consensys team continues to focus its roadmap on performance, stability, modularity, and upcoming protocol changes like the Verkle Trie migration. The Besu Maintainers intend to support the Prague hard fork and take an active role in its delivery. 
```

Comment:
Would be good to add a mention of the focuses from the other contributing organizations (Citi, Kaleido, Web3Labs) here, which I assume would be leaning towards the enterprise features.